### DISCUSSION TITLE
CarPlay - Audiobook Resumes After Interrupt

### DISCUSSION THREAD

**Discussion by mandopatriot** [LeoKlaus/plappa#50] [Feature Requests]
This feature request is to have Plappa resume playing the audiobook after an interrupt by Siri during CarPlay. For example, during navigation steps/alerts or searching, receiving/sending messages, etc. Currently playback stops and has to be manually resumed on version 1.1.1, iOS 17.3.1.

**Comment by LeoKlaus** [comment]
Wow, that's annoying. I though this happened automatically, but I'll definitely take care of this!

**Comment by LeoKlaus** [comment]
Just got it working, will be fixed with the next update!
